.github/workflows/haskell.yml

```diff
@@ -31,14 +30,30 @@ jobs:
         path: ~/.cabal
```

Comment:
does this path works for windows? The last build is not using cache but it doesn't save it afaics

Reply:
The path is working, I think it's this issue https://github.com/actions/cache/issues/144